tests/5.1/dispatch/test_dispatch_depend.c

```diff
@@ -0,0 +1,66 @@
+//===---- test_dispatch_depend.c -----------------------------------------===//
+// 
+// OpenMP API Version 5.1
+//
+// Uses dispatch construct as context for variant directive. Uses depend clause
+// which adds depend property to the interoperability requirement set.
+//
+// Inspired by "OpenMP 5.1 Features: The Dispatch Construct" video:
+// https://www.youtube.com/watch?v=ruugaX95gIs
+// 
+//===-------------------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include "ompvv.h"
+
+#define N 1024
+
+int arr[N]; // implicit map array 
```

Comment:
The comment does not really make sense as there is no target region.
```suggestion
int arr[N];
```